### PR TITLE
Decouple Shareable from DayLimit

### PR DESCRIPTION
**Pull request by adklempner**

### CODE REVIEW

File: contracts/DayLimit.sol [hunk 1]
```diff
@@ -38,13 +38,13 @@ contract DayLimit is Shareable {
...
+  function setDailyLimit(uint _newLimit) external {
     dailyLimit = _newLimit;
   }
 
-  // resets the amount already spent today. needs many of the owners to confirm
-  function resetSpentToday() onlymanyowners(sha3(msg.data)) external {
+  // resets the amount already spent today.
+  function resetSpentToday() external {
     spentToday = 0;
   }
```
Comment (maraoz) [inline contracts/DayLimit.sol]: I don't know if `resetSpentToday` and `setDailyLimit` make sense without authentication. Maybe we should make them internal?

Comment (adklempner) [inline contracts/DayLimit.sol]: Making the two functions internal here means they also have to be internal when overriding them in `MultisigWallet`. Should they be internal in `MultisigWallet` as well?

Comment (maraoz) [inline contracts/DayLimit.sol]: Ahh, really? That's inconvenient. They shouldn't be internal in MultisigWallet. Can you check if there's a way of makin them external?

Comment (adklempner) [inline contracts/DayLimit.sol]: The solidity compiler requires overloaded functions to have the same extended function signature, so the only way to make them internal in DailyLimit and external in MultisigWallet is to give them different names.

**Pull request merged**